### PR TITLE
Added Calendar api

### PR DESCRIPTION
**Pull request by Pawan2061**

### CODE REVIEW

**Comment by KaneshiroKeola** [comment]
I can add a Calender Api

**Comment by devsargam** [comment]
@KaneshiroKeola can you add some other API since @Pawan2061 has already added calender API

File: api/calendar.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Calendar API
```
Comment (devsargam) [inline api/calendar.md]: It seems the API gives information about Holidays so rename `Calender` to `Holidays`
```suggestion
name: Holidays API
```

File: api/calendar.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+categories:
+  - calendar
+url: https://www.abstractapi.com/api/holidays-api
+description: Retrieve religious, non-public and public holidays for 200+ countries worldwide and for any specific year
```
Comment (devsargam) [inline api/calendar.md]:
```suggestion
description: Retrieve holidays for 200+ countries
```

**Review by devsargam — CHANGES_REQUESTED** [review]

**Comment by devsargam** [comment]
@Pawan2061 can you work on the following???

**Pull request merged**